libs/ipc/inc/public/tfc/ipc/details/filter.hpp

```diff
@@ -247,6 +255,11 @@ struct any_filter_decl<std::string> {
   using value_t = std::string;
   using type = std::variant<filter<filter_e::filter_out, value_t>>;
 };
+template <>
+struct any_filter_decl<ipc::details::mass_t> {
+  using value_t = ipc::details::mass_t;
+  using type = std::variant<filter<filter_e::filter_out, value_t>, filter<filter_e::offset, value_t>>;
```

Comment:
like said in comment I do not like the offset being an option here

Reply:
I'll remove it, don't have a strong opinion on this.